devon4j-app/index.asciidoc

```diff
@@ -76,14 +84,23 @@ buildJava("sampleapp", false)
 Once build is successful you will get bootified-war generated in server module target folder. In sampleapp check for path sampleapp/server/sampleapp-server-bootified.war
 
 ====
+
+
 ====
 ## Conclusion
 
 In this tutorial you have learnt how to create devon4j application using command line or via Eclipse. Next you can check for other tutorials such as:
+
  * Java backend code generation using CobiGen (Code Generator)
+ 
```

Comment:
When the problem with CobiGen is fixed and this tutorial is working again, then we can link it here. So nothing to do at the moment, but keep in mind to link other existiing tutorials from here

Reply:
As of now I have removed cobigen link and modified the document.